lib/ui/viewer/gallery/component/group/lazy_group_gallery.dart

```diff
@@ -75,6 +76,7 @@ class _LazyGroupGalleryState extends State<LazyGroupGallery> {
     _logger = Logger("LazyLoading_${widget.logTag}");
     _shouldRender = true;
     _files = widget.files;
+    _filesAsSet = _files.toSet();
```

Comment:
Instead of conversion during selection, this will do it during load. May be we can make it a lazy getter